sales.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE>
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Salmon Cookies</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <section id="">
+      <!-- put this in the main page not the sales page... sales page should only have info for the owner...LINES:10-16 -->
+      <h1 id="title">--------------  Salmon Cookies  --------------</h1>
+      <img src="project-pictures/salmon.jpg" class="center"/>
+        <p id="cookieExp">Salmon cookies are cookies made into the shape of a salmon, and actually made with salmon (though the fish is ground up so fine that you can't even taste it!) that is the ideal match for a steaming cup of coffee on a gray Seattle morning. Or gray Seattle afternoon... </p>
```

Comment:
Can you think of a cleaner way to format this paragraph? Also, since this p tag is a sibling of the img tag above it, they should be indented at the same level.